src/util/statusSeverity.ts

```diff
@@ -0,0 +1,37 @@
+import { Protocol } from '../protocol/protocol';
+
+export class StatusSeverity {
+
+    private status: Protocol.Status;
+
+    constructor(status: Protocol.Status) {
+        this.status = status;
+    }
+
+    public isOk(): boolean {
+        return this.status
+          && this.status.severity === 0;
```

Comment:
This could be done using testSeverity as well, right? :)

Reply:
@jrichter1 I'd think that ok is when there's no flag at all, so I need to use ```===```, right?